src/base.css

```diff
@@ -12,6 +12,8 @@ body {
 
 * {
   box-sizing: border-box;
+  scrollbar-width: thin;
+  scrollbar-color: transparent;
```

Comment:
I wish there was a better way to scope this than `*`* but I don't suppose there is.